packages/client/src/app/core/services/http-request.service.js

```diff
@@ -108,6 +108,11 @@ export class HttpRequestService {
         }
       : errorResponse.response.data;
 
+    if (error.error === 'ERR_NETWORK') {
+      error.error = error.message;
+      error.message =
+        'Er is iets mis gegaan met het versturen van je request.\nProbeer het later nog eens.';
```

Comment:
I would use the word verzoek instead of request

Reply:
Sounds indeed more natural